keyboards/idobao/id61/info.json

```diff
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "IDOBAO",
+    "keyboard_name": "ID61",
+    "maintainer": "vinorodrigues",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "debounce": 5,
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "backlight": false,
+        "rgblight": false
+    },
+    "matrix_pins": {
+        "cols": ["C7", "F6", "F5", "F4", "F1", "B7", "D5", "D1", "D2", "D3", "D4", "D0", "D6", "D7"],
+        "rows": ["B0", "B1", "B2", "B3", "F7"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://idobao.net/search?type=product&q=id61*",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0061",
+        "vid": "0x6964"
+    },
+    "layouts": {
```

Comment:
```suggestion
    "community_layouts": ["60_ansi"],
    "layouts": {
```

Reply:
Moving to community layout has issues since `brandonschlack-ansi`, `mechmerlin-ansi`, & `talljoe-ansi` do not compile.

Reply:
That's fine, they're not your problem to solve.

Reply:
ok.  committed as requested.